web/src/app/services/navigation/navigation.service.ts

```diff
@@ -236,6 +237,11 @@ export class NavigationService {
     this.router.navigateByUrl(url, {replaceUrl: true});
   }
 
+  navigateToEditSurvey(surveyId: string): void {
+    const url = `${NavigationService.SURVEYS_SEGMENT}/${surveyId}/${NavigationService.SURVEYS_EDIT}`;
```

Comment:
Nit: Maybe we should create a utility file that we can share between the navigation service and the route configs to ensure that the routes defined here and there are consistent.

Reply:
Sg, added a test.